services/wallet/token/token.go

```diff
@@ -435,13 +435,14 @@ func (tm *Manager) GetAllTokens() ([]*tokenTypes.Token, error) {
 	allTokens = append(uniqueListsTokens, allTokens...)
 
 	overrideTokensInPlace(tm.networkManager.GetEmbeddedNetworks(), allTokens)
+	cleanTokens := removeNativeTokens(allTokens)
```

Comment:
@Khushboo-dev-cpp this should be done on the `tokenlists` package level. Please move this to `rebuildTokensMap` function and call it before the `t.solveCollision()` call.